integration/nwo/network.go

```diff
@@ -1439,6 +1461,79 @@ func (n *Network) peerCommand(command Command, tlsDir string, env ...string) *ex
 	return cmd
 }
 
+func (n *Network) downloadOldBinaries(fabricVersion string) error {
+	if _, err := os.Stat(filepath.Join(n.RootDir, "bin")); errors.Is(err, os.ErrNotExist) {
+		resp, err := http.Get(
+			fmt.Sprintf("https://github.com/hyperledger/fabric/releases/download/v%s/hyperledger-fabric-%s-amd64-%s.tar.gz", fabricVersion, runtime.GOOS, fabricVersion))
+		defer resp.Body.Close()
+
+		if err != nil {
+			return fmt.Errorf("could not dowload fabric binaries")
+		}
+
+		// Create the file
+		out, err := os.Create(n.RootDir + "/binaries.tar.gz")
+		if err != nil {
+			return fmt.Errorf("could not create destination dowload file for fabric binaries")
+		}
+		defer out.Close()
+
+		// Write the body to file
+		_, err = io.Copy(out, resp.Body)
+		if err != nil {
+			return fmt.Errorf("could not dowload fabric binaries")
+		}
+
+		tarCmd := exec.Command("tar", "-xf", out.Name(), "--directory", n.RootDir)
+		err = tarCmd.Run()
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (n *Network) oldPeerCommand(command Command, fabricVersion string, tlsDir string, env ...string) *exec.Cmd {
```

Comment:
why do you need to have this `oldPeerCommand` command?

Reply:
In the integration test integration/lifecycle/capabilities_endorse_test.go I upgrade the channel capabilities and intend to show that a peer that does have source code to support it won't endorse any more transactions. Therefore, one of the peers runs with a fabric v2.4.5 binary, downloaded from Github.

Reply:
did you mean to say "does not" instead of "does" ? I still don't understand, why would the peer not endorse transactions? If you enable a capability and a peer cannot support it, it simply crashes.

Reply:
I assumed that the peer crash should be proven. If you consider this verification unnecessary I can remove it.

Reply:
yes please remove